src/navbar/navbar.html

```diff
@@ -34,6 +34,24 @@
         <div click.delegate="navigate('/launches')" class="item ${(router.currentInstruction.config.name === 'launches') ? 'active' : ''}">Launches</div>
         <div click.delegate="navigate('/documentation')" class="item ${(router.currentInstruction.config.name === 'documentation') ? 'active' : ''}">Documentation</div>
         <div click.delegate="goto('https://primedao.typeform.com/PrimeLaunchSup')" class="item">Support</div>
+
+        <!-- <div click.delegate="goto('https://primedao.typeform.com/PrimeLaunchSup')" class="item">Support</div> -->
+      </div>
+      <div class="dev-go-to-seed-stage6">
```

Comment:
restricted to our dev account, because it is helpful for Celo mainnet.
wdyt?

Reply:
I'd say restrict only to development environment, but not for specific wallet, as it might be useful for testing with different wallets (?)

`process.env.NODE_ENV === 'development'`